src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -405,16 +472,17 @@ Example::
 
    rsqrt([4,9,16]) = [0.5, 0.33333334, 0.25]
 
+The storage type of ``rsqrt`` output is always dense
+
 )code" ADD_FILELINE)
-.set_attr<FCompute>("FCompute<cpu>", UnaryCompute<cpu, mshadow_op::reciprocal_square_root>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_rsqrt"});
 
-MXNET_OPERATOR_REGISTER_BINARY(_backward_rsqrt)
-.set_attr<FCompute>("FCompute<cpu>",
-  BinaryCompute<cpu, unary_bwd<mshadow_op::reciprocal_square_root_grad> >);
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(_backward_rsqrt,
+                                             unary_bwd<mshadow_op::reciprocal_square_root_grad>);
 
 // exp
-MXNET_OPERATOR_REGISTER_UNARY(exp)
+MXNET_OPERATOR_REGISTER_UNARY_COMPUTE_DR(exp, cpu, mshadow_op::exp)
```

Comment:
Actually, this isn't entirely the case -- the backwards passes are, in fact, not "fallbacks"